plugins/woocommerce-admin/client/task-lists/fills/tax/woocommerce-tax/setup.tsx

```diff
@@ -91,6 +96,10 @@ export const Setup: React.FC< SetupProps > = ( {
 		pluginsToActivate,
 	};
 
+	const prefetchedJetpackAuthData = useSelect( ( select ) => {
```

Comment:
Prefetching the Jetpack auth URL here and then passing it to the `<Connect>` component so users don't have to wait for the "Connect" to finish retrieving the auth URL from the REST API.